src/Moryx.Runtime.Wcf/Moryx.Runtime.Wcf.csproj

```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>WebService integration for MORYX applications based on WCF</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>webservice;web;wcf;communication;castle;moryx;iiot;iot</PackageTags>
```

Comment:
This class library is used for wcf integration into runtime environment

maybe: "MORYX;Runtime;Server;Client;WCF;communication"